scripts/bundle.mjs

```diff
@@ -0,0 +1,134 @@
+import esbuild from 'esbuild';
+import { readFileSync, writeFileSync } from "fs";
+import { join, relative } from "path";
+import { cwd } from "./utils.mjs";
+const CWD = cwd(import.meta.url);
+const ROOT = join(CWD, '..');
+const SHOW_METAFILE = process.argv.some(string => string === '--metafile');
+
+/**
+ * Deliberately not try-catching around these operations.
+ * Node's errors around paths etc. are descriptive and anything that
+ * fails here should fail the process.
+ */
+(async () => {
+    // default build
+    await bundle({});
+
+    // debug build
+    await bundle({
+        plugins: [],
+        outfile: join(ROOT, 'dist/autofill-debug.js')
+    });
```

Comment:
there's no speed difference to run these in parallel, so just let them be sequential